lib/commands/clipboard.js

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'teen_process';
+import log from '../logger';
+
+const commands = {};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {!string} content - The content to be set as base64 encoded string.
+ * @param {?string} contentType [plaintext] - The type of the content to set.
+ *                                            Only `plaintext` and 'image' are supported.
+ */
+commands.setClipboard = async function setClipboard (content, contentType) {
+  if (contentType === 'plaintext') {
+    return await exec('powershell', ['-command', `$str=[System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String(${content}));Set-Clipboard -Value $str`]);
+  } else if (contentType === 'image') {
+    return await exec('powershell', ['-command', `$img=[Drawing.Bitmap]::FromStream([IO.MemoryStream][Convert]::FromBase64String(${content}));[System.Windows.Forms.Clipboard]::SetImage($img);$img.Dispose();`]);
+  }
+  throw new log.Error(`Could not set clipboard as ${contentType} is not a supported content type.`);
```

Comment:
there is no log.Error object. Simply `Error`

Reply:
or use `log.errorAndThrow` method

Reply:
Yes - I was using log.errorAndThrow originally, then decided to use the native Error but didn't check before pushing and ended up with a combo of the two... sorry!